SITCOMTN-023.tex

```diff
@@ -49,13 +48,129 @@
+\item{Story/Bug}: Epic Link, Sprint, Story Points, Supported Epic, Related Issues, Needs to be done before/after
+\item{Tasks}: Story Points, Needs to be dong before/after
+\end{itemize}
+
+\subsection{Workflows}
+There will be two different workflows, depending on if the ticket type is a milestone or not, as shown in Figure \ref{fig:workflow}. The main differences are the acknowledged and invalid states available in the non-milestone case.  This comes from the nature of the milestones ticket being only an achieved goal instead of an activity that needs to be done. 
+
+\begin{figure}[h]
+\begin{center}
+\includegraphics[width=0.48\textwidth]{WorkFlowMilestones.png}
+\includegraphics[width=0.48\textwidth]{WorkFlowNonMilestones.png}
+\caption{\label{fig:workflow} Workflows}
+\end{center}
+\end{figure}
+
+
+\subsection{Ticket Creation and Review process}
+In order to enforce a common process, the workflow has constraints on who is allowed to create tickets and move them to different state transition. 
+
+Only SIT-COM leads can create, deprecate and/or reopen Milestones tickets. SCLT coordinators and leads can create an Epic. However, only the assignee can invalidate, deprecate, reopen and/or complete the Epic. Stories/bugs are created by a SIT-Com member and the tasks by anyone on the project. 
+
+One particularity of the non-milestone workflow is the required acknowledgement status, only allowed by the assignee, ensuring that the assignee has received the tasks and will act on it. As expected, the assignee as well as the start and end dates are required to transition to the acknowledge state.
+
+Regarding the review process, reviewers must be populated before submitting the ticket for Review as an email will automatically be sent to all reviewers as part of the transition. All reviewers must approve ticket in order for the ticket to transition to Reviewed status, which is done automatically after the approval from the final reviewer. All child and/or included tickets must be approved first. 
+In case of disapproval, the system will automatically put the ticket back into In Progress status and only the assignee can send it back to the In Review state. Each reviewer is required to add a comment as part of the review process, independently of the outcome. 
+Note that the ticket can not be edited while in In Review, Reviewed state.
+
+
+Table \ref{tab:owner} summarizes the ownership of the creation and review process of each SIT-Com Jira ticket. 
+\begin{table}
+\begin{center}
+\caption{\label{tab:owner} Summary of Ownership}
+\begin{tabular}{|p{1.8cm}|p{1.8cm}|p{1.8cm}|p{1.8cm}|p{1.8cm}|p{1.8cm}}
+\hline
+Ticket type vs Use Case              & Milestone 2	& Milestone 3& Epics	& Story/Bug	& Tasks \\
+\hline
+\hline
+Milestone Tracking & {\bf Creator}: P6 &{\bf Creator}: P6& {\bf Creator}: One SCLT Coordinator & {\bf Creator}: SIT-Com member & {\bf Creator}: Anyone \\
```

Comment:
Jira uses the term "Reporter"  but since you are somewhat talking about where the tickets come from maybe "origin" is appropriate. "Creator" sounds like a supreme being :)